Makefile

```
@@ -10,6 +12,16 @@ build: $(go_sources)
 	mkdir -p build
 	CGO_ENABLED=0 go build -o build/foreman_worker $^
 
+.PHONY: data
+data: build/data/com.redhat.Yggdrasil1.Worker1.foreman.conf build/data/com.redhat.Yggdrasil1.Worker1.foreman.service
+
+.PHONY: install
+install: build data
+	install -D -m 755 build/foreman_worker $(DESTDIR)/$(LIBEXECDIR)/foreman_worker
```

`LIBEXECDIR` already starts with `/` so this is redundant
```suggestion
	install -D -m 755 build/foreman_worker $(DESTDIR)$(LIBEXECDIR)/foreman_worker
```